packages/svelte/src/compiler/compile/index.js

```diff
@@ -30,7 +30,8 @@ const valid_options = [
 	'loopGuardTimeout',
 	'preserveComments',
 	'preserveWhitespace',
-	'cssHash'
+	'cssHash',
+	'disableVersionTagging'
```

Comment:
also needs enhancement of the types

Reply:
https://github.com/sveltejs/svelte/pull/8761/files#diff-7701b34eb01df7929afb01e19ffa73386489113824d279f81b1400e39aa757ceR198

I updated the CompileOptions interface, are there any other places that need to be changed?